packages/inter-protocol/src/auction/auctioneer.js

```diff
@@ -54,6 +55,14 @@ const makeBPRatio = (rate, currencyBrand, collateralBrand = currencyBrand) =>
     AmountMath.make(collateralBrand, BASIS_POINTS),
   );
 
+const makeTransferNonEmpty = transfers => {
```

Comment:
I don't follow why this is necessary. Won't empty transfers be no-ops?

Reply:
I think it's net negative for legibility. I understand why a test would want to ignore empty allocations; for that it would be better have a helper to filter out empty allocations in the test. I'm not as sure that the end user would benefit from omitting empty allocations.

I request that:
- this concern be removed from the contract for now
- when we move atomic transfer capability into ZCF we revisit whether/how to omit empties in production

No opinion on whether the tests validate empties or ignore them.

Reply:
Reverted.